libs/ui/src/theme/theme.ts

```diff
@@ -74,6 +72,17 @@ export const defaultDarkTheme: DefaultTheme = {
   ...greenDark,
   ...yellowDark,
   ...redDark,
+  field: {
+    fontWeight: font.weight.reg,
+    fontSize: font.size.md,
+    size: {
+      md: '28rem',
+      lg: '52rem',
+      full: '100%',
+    },
+    radius: border.radius,
```

Comment:
Maybe this and delete the other file. You would have to update the hard-coded value everywhere so maybe not.
```suggestion
    radius: '0.4rem',
```

Reply:
Yeah wanted to put in a spot to easily change in the repo. kinda weak that is only 1 variable in that file.